server/express_app.js

```diff
@@ -36,7 +36,6 @@ if (config.get('debug')) {
 }
 
 app.use('/v' + version.getApiVersion(), router);
-app.use('/', (new express.Router()).get('/', router.generateIndexPage));
```

Comment:
It makes it so that `/v1` as well as `/` generate the index page. Otherwise `/` returns "Not Found"